handler.js

```diff
@@ -714,6 +715,10 @@ class Pr extends GithubBuild{
     return envVariables;
   }
 
+  enableUatAndFunctionalTests() {
+    return this.event.pull_request.base.ref === "release";
```

Comment:
Sorry @tuanmh  for removing that function. I reused it in a different way to detect `[on push|pr]` command.

Reply:
No probs at all @mpgo13, I didn't review it carefully, it's good if we could refactor this file into multiple files - otherwise, it would be hard for peer review!